src/overlays/actors/ovl_En_Tag_Obj/z_en_tag_obj.c

```diff
@@ -20,10 +25,22 @@ const ActorInit En_Tag_Obj_InitVars = {
     (ActorFunc)EnTagObj_Update,
     (ActorFunc)NULL
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Tag_Obj_0x80B12870/EnTagObj_Init.asm")
+void EnTagObj_Init(Actor* thisx, GlobalContext* globalCtx) {
```

Comment:
As discussed, i think this is a bit cleaner and it's OK as well.
![image](https://user-images.githubusercontent.com/13141469/112303742-ee8df680-8c9c-11eb-92fa-310dfdcb1316.png)
Due to this, its more of a stylistic choice, temp or no temp.
Just as reminder since you didn't respond to that on discord.
This isn't a stopper for this PR